ui/src/components/wallet-button/wallet-button.tsx

```diff
@@ -41,15 +41,15 @@ const WalletMenu: React.FC = () => {
           _hover={{ bg: 'custom.gray.100' }}
           bg={'custom.gray.200'}
           onClick={handleCopyAccount}
-          icon={<Icon name="copy" />}
+          // icon={<Icon name="copy" />}

```

Comment:
Remove all the comments if they will not be used anymore. In it will be kept in the future, leave a TODO comment with why.